src/gui/src/layoutViewer.cpp

```diff
@@ -2468,7 +2486,8 @@ void LayoutViewer::updateBlockPainting(const QRect& area)
   repaint_requested_ = false;
 
   // build new drawing of layout
-  auto* block_drawing = new QPixmap(area.width(), area.height());
+  auto* block_drawing = new QImage(area.width(), area.height(),
```

Comment:
Have you measured that this is faster?  
"QImage is designed and optimized for I/O, and for direct pixel access and manipulation, while [QPixmap](https://doc.qt.io/qt-5/qpixmap.html) is designed and optimized for showing images on screen."

Reply:
20% reduction in frametime.